modules/configuring-an-authentication-provider-in-central.adoc

```diff
@@ -0,0 +1,64 @@
+[id="configuring-an-authentication-provider-in-central_{context}"]
+= Configuring an authentication provider in Central
+
+[role="_abstract"]
+You can configure an authentication provider in Central and initiate the login process with the `roxctl` CLI.
+Set the variable `ROX _ENDPOINT`, initiate the login process with the `roxctl central login` command, select the authentication provider in a browser window, and retrieve the token information from the `roxctl` CLI.
+
+.Procedure
+
+. Select an authentication provider of your choice, such as OpenID Connect (OIDC) with fragment or query mode, and configure it in Central.
+
+. Run the following command to set the `ROX _ENDPOINT` variable to Central hostname and port:
++
+[source,terminal,subs="+quotes"]
+----
+export ROX_ENDPOINT=_<central_hostname:port>_
+----
+
+. Run the following command to initiate the login process to Central:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl central login
+----
+
+. Within `roxctl` CLI, a URL is printed as output and you are redirected to a browser window where you can select the authentication provider you want to use.
+
+. Once you have successfully logged in with the authentication provider, the browser window indicates that authentication was successful and you can close the browser window.
+
+. `roxctl` CLI prints your token information. This includes details such as the access token, the expiration time of the access token, the refresh token if one has been issued, and notification that these values are stored locally.
++
+.Example output
++
+[source,terminal]
+----
+Please complete the authorization flow in the browser with an auth provider of your choice.
+If no browser window opens, please click on the following URL:
+        http://127.0.0.1:xxxxx/login
+
+INFO:	Received the following after the authorization flow from Central:
+INFO:	Access token: <redacted> <1>
+INFO:	Access token expiration: 2023-04-19 13:58:43 +0000 UTC <2>
+INFO:	Refresh token: <redacted> <3>
+INFO:	Storing these values under $HOME/.roxctl/login… <4>
+----
+<1> The access token.
+<2> The expiration time of the access token.
+<3> The refresh token.
+<4> The directory where values of the access token, the access token expiration time, and the refresh token are stored locally.
++
+[IMPORTANT]
+====
+Ensure that you set the environment variables according to your preferences to determine the directory where the configuration is stored.
+
+* If you set the `$ROX_CONFIG_DIR` environment variable, the configuration is stored in the `$ROX_CONFIG_DIR/roxctl-config` directory. This option has the highest priority.
+
+* If you set the `$XDG_RUNTIME_DIR` environment variable and the `$ROX_CONFIG_DIR` variable is not set, the configuration is stored in the `$XDG_RUNTIME_DIR /roxctl-config` directory.
+
+* If you do not set the `$ROX_CONFIG_DIR` or `$XDG_RUNTIME_DIR` environment variable, the configuration is stored in the `$HOME/.roxctl/roxctl-config` directory.
+====
```

Comment:
When do you set this environment variable? Can the conditions in the third bullet be stated as default? Might be less confusing. 

> ...to determine the directory where the configuration is stored, by default `$HOME/.roxctl/roxctl-config`.

Reply:
I suppose one would set those environment variables within their setup, so this might be something they have in their shell environment under `etc/profile` or `.bashrc`.

Ultimately, the default option is the `$HOME/.roxctl/roxctl-config` one, so we can state this as default.